core/health/shell/src/main/java/org/opennms/core/health/shell/MetricsListCommand.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2018 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2018 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.core.health.shell;
+
+import java.util.List;
+
+import org.apache.karaf.shell.api.action.Action;
+import org.apache.karaf.shell.api.action.Command;
+import org.apache.karaf.shell.api.action.lifecycle.Reference;
+import org.apache.karaf.shell.api.action.lifecycle.Service;
+import org.osgi.framework.BundleContext;
+
+@Command(scope = "health", name = "metrics-list", description="List the available metric sets.")
+@Service
+public class MetricsListCommand implements Action {
+
+    @Reference
+    private BundleContext bundleContext;
+
+    @Override
+    public Object execute() {
+        final List<NamedMetricSet> metricSets = NamedMetricSet.getNamedMetricSetsInContext(bundleContext);
+        if (metricSets.size() < 1) {
+            System.out.println("(No metric sets are currently available.)");
+            return null;
+        }
+
+        for (NamedMetricSet namedMetricSet : metricSets) {
+            System.out.printf("Metric set: %s\n", namedMetricSet.getName());
+            if (namedMetricSet.hasDescription()) {
+                System.out.printf("\t%s\n", namedMetricSet.getDescription());
+            }
+        }
+        System.out.println();
```

Comment:
Can we print the metrics in one line instead of splitting them? 
I would imagine something like this:

```
Name       Description
Flows      Description of metric
...        ...
```